pyocd/__main__.py

```diff
@@ -691,6 +693,8 @@ def do_gdbserver(self):
                 if self._args.elf:
                     session.board.target.elf = os.path.expanduser(self._args.elf)
                 for core_number, core in session.board.target.cores.items():
+                    if isinstance(session.board.target.cores[core_number], GenericMemAP):
```

Comment:
Could you please add a comment here just mentioning why the GenericMemAP is skipped? It's obvious to us, but not necessarily other people. (Same for pyocd/tools/gdb_server.py.)